Money_market_account.rb

```diff
@@ -0,0 +1,21 @@
+module Bank
+  require 'pry'
+
+  class MoneyMarketAccount < Account
+    attr_reader :checks, :transactions
+
+    def initialize (account_id, balance, open_date, owner = nil)
+      super(account_id, balance, open_date, owner = nil)
+      @withdrawalfee = 1
+      @min_initial_balance = 10000
+      @transactions = 6
```

Comment:
Since you use the max transactions throughout, this might be a good place to use a constant variable